model-per-cluster/readme.md

```diff
@@ -24,11 +24,23 @@ make predictions.
 
 ## How to install
 
+### Using bigmler
+
+If you have bigmler installed in your system, just checkout this
+repository and, at its top level, issue the command:
+
+        make compile PKG_DIR=model-per-cluster
+
+That will create for you all necessary libraries and the three
+scripts.
```

Comment:
Similar idiomatic change: "That will create all necessary libraries and the three scripts for you."